tests/test_read_only_db.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Tyler Neely
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+extern crate libc;
+extern crate rocksdb;
```

Comment:
Have you switched to Rust 2018 edition ? It looks like redundant.

Reply:
No, I haven't made that switch yet. I know we had some hesitancy to making the switch until it had been released a while (see #224).  I think we should before our next release, but I don't want to take that on as part of this PR as well. I'm happy to start another PR that makes the switch after this one is merged.